### PR TITLE
read me edit

### PR DESCRIPTION
**Pull request by humulla**
Link to JIRA ticket
https://hackney.atlassian.net/jira/software/projects/PA/boards/35?selectedIssue=PA-340

What changes have we introduced
I have made changes to the readMe file so it links to Contact API

### CODE REVIEW

File: ResidentContactApi/V1/Controllers/ResidentContactApiController.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ public IActionResult ListContacts([FromQuery] ResidentQueryParam rqp)
         /// </summary>
         /// <response code="200">Successful operation</response>
         /// <response code="404">Contact not found for specified ID</response>
-        /// <response code = "400">Please enter a valide request.</response>
+        /// <response code = "400">Please enter a valid request.</response>
```
Comment (Renny4Real) [inline ResidentContactApi/V1/Controllers/ResidentContactApiController.cs:48]: Nice spot 🎉 !

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 Resident Contact API will return a list of contact details (inc. mobile, uprn, email) for residents that we have stored.
```
Comment (Renny4Real) [inline README.md:3]: This feels more like a description for the LIST endpoint rather than the API

File: README.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
 
+The contact details can be retrieved through the use of a GET Endpoint which will return all the data or a specific Id too.
```
Comment (Renny4Real) [inline README.md:5]:
```suggestion
The contact details can be retrieved through the use of two GET Endpoints. One will return all the contact detail records stored, the other will return a single record for specified ID.
```

File: README.md (hunk 2 above)
Comment (humulla) [reply README.md:3]: I wanted opinions on this, do you think I should remove it, it was already there so wasn't too sure

Comment (Renny4Real) [reply README.md:3]: I think you could rephrase it to describe the API. Maybe something like:
```suggestion
Resident Contact API allows access to saved contact details (inc. mobile, uprn, email) for residents that we have stored.
```

**Review by Renny4Real — APPROVED** [review]
